rfcs/0000-template.md

```diff
@@ -0,0 +1,51 @@
+* Start Date: (Use today's date in YYYY-MM-DD format)
+* RFC PR: (Leave empty till PR Created)
+
+# Summary
+
+A brief explanation of the proposed feature.
+
+# Basic Example
+
+If the proposal involves a new or changed component, includ a basic code example. Omit this section if it's not applicable.
+
+# Motivation
+
+Why are you proposing this? What use cases does it support? What are your expected outcomes.
+
+Please focus on explaining the motivation so that if this RFC is not accepted, the motivation could be used to develop an alternative solution. In other words, enumerate the constraints you are trying to solve without coupling them too closely to the solution you have in mind.
+
+# Detailed Proposal
+
+This is the bulk of the RFC. Explain the component and the design in enough detail for someone familiar with design systems and web components can understand, and someone familiar with the implementation help to implement it. This should get into as many specifics as possible, outline any edge cases or extra consideration, and include examples of how it can be used. Any new terminology should also be defined here.
+
+# Drawbacks
+
+Why should the community *not* do this? Please consider:
+* Implementation cost, both in terms of code size and complexity.
+* Whether the proposed feature can be implemented in the user space
+* The impact of the feature on people learning OpenUI's Design System
+* Integration of the feature with others that exist and the impact on any compatibility with existing features.
+* Migration implications for current users of OpenUI's Design System
+
+There are tradeoffs to choosing any path. Attempt to identify them here.
+
+# Alternatives
+
+What other designs or features have been considered? What is the impact of this not being included.
```

Comment:
Add in here, does a solution already exist within OpenUI, if so what makes this solution different?